core/src/tangram.cpp

```diff
@@ -121,15 +121,17 @@ namespace Tangram {
                 Label::s_needUpdate = false;
 
                 for (const auto& mapIDandTile : m_tileManager->getVisibleTiles()) {
-                    const std::shared_ptr<MapTile>& tile = mapIDandTile.second;
-                    tile->update(_dt, *m_view);
+                    const auto& tile = mapIDandTile.second;
+                    if (tile->isReady())
```

Comment:
Styling nitpick.
@hjanetzek we have been using explicit {} for single line if blocks.